commands/create.go

```diff
@@ -135,15 +138,14 @@ func runCreate(dockerCli command.Cli, in createOptions, args []string) error {
 		}
 	} else {
 		if len(args) > 0 {
-			ep, err = validateEndpoint(dockerCli, args[0])
+			ep, err = dockerutil.ValidateEndpoint(dockerCli, args[0])
```

Comment:
let's not make unnecessary new public api. This function is called in exactly one place. If you want to move some function definition to separate files then that is fine(but should probably be in a separate refactoring PR).